### ISSUE TITLE
Comic text shouldn't be a stop for each word

### ISSUE THREAD

**Issue by livcheerful** [livcheerful/anywhereadventures#49]
Each bubble should have one stop

**Comment by livcheerful** [comment]
Fixed. redesign comic component
